extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -718,8 +726,16 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 		this._apiVersion = API.defaultVersion;
 		this._tsserverVersion = undefined;
 	}
-}
 
+	public configurePlugin(pluginName: string, configuration: any, reconfigureOnRestart?: boolean): any {
```

Comment:
Scope this to TS 3.2:

```ts
if (this._apiVersion.gte(API.v320)) {
    ...
}
```

Not sure if it should also cause the command to fail. Do you need this information on your side?